Vwitter/UserCell.h

```diff
@@ -0,0 +1,20 @@
+//
+//  UserCell.h
+//  Vwitter
+//
+//  Created by Christina Li on 7/11/22.
+//
+
+#import <UIKit/UIKit.h>
+#import <Parse/Parse.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface UserCell : UITableViewCell
+@property (strong, nonatomic) PFUser *user;
+@property (weak, nonatomic) IBOutlet UILabel *username;
```

Comment:
nit: it should always be clear based off of the property name what something is. In this case you are doing a good job of this with `followStatusButton` (it is clear that it's a button). But for `username`, it would be better to call it `usernameLabel` to make it clear that it's a label (UI element)